### ISSUE TITLE
balena scan --json output has terminal escape sequences before JSON content

### ISSUE THREAD

**Issue by sgrimm** [balena-io/balena-cli#2098]
# Expected Behavior

I should be able to pipe `balena scan --json` to something that expects JSON input. For example, `sudo balena scan --json | jq .` should output a jq-formatted copy of the scan results.

# Actual Behavior

```
$ sudo balena scan --json | jq .
parse error: Invalid numeric literal at line 1, column 2
```

This appears to be because `balena scan --json` outputs some terminal escape sequences before the JSON.

```
$ sudo balena scan --json | od -tc | head -1
0000000  033   [   2   K 033   [   1   G  \n   [  \n                   {
```

# Steps to Reproduce the Problem

With at least one device on the local network in local mode, run `sudo balena scan --json | jq .`

# Specifications

- **balena CLI version:** balena-cli version "12.25.0", Node.js version "12.16.2"
- **Cloud backend:** balenaCloud
- **Operating system version:** macOS 10.15.7
- **32/64 bit OS and processor:** 64-bit Intel
- **Install method:** Homebrew

**Comment by pdcastro** [comment]
Thanks for reporting this issue @sgrimm 👍 
Pull request #2099 should fix it (after it is merged, pending review).

**Comment by pdcastro** [comment]
Fixed in CLI v12.28.1. (Spinner animation goes to stderr, JSON output goes to stdout.)
